prisma/prisma.ts

```diff
@@ -5,10 +5,11 @@ let prisma: PrismaClient;
 if (process.env.NODE_ENV === 'production') {
   prisma = new PrismaClient();
 } else {
-  if (!global.prisma) {
-    global.prisma = new PrismaClient();
+  if (!global) throw new Error('Global object not found.');
+  if (!(global as Record<string, any>).prisma) {
+    (global as Record<string, any>).prisma = new PrismaClient();
   }
-  prisma = global.prisma;
+  prisma = (global as Record<string, any>).prisma;
```

Comment:
This is objectively bad, but I am not currently knowledgeable enough to figure out a better method.